route/account/_account-id/token/_token-id/index.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+module.exports = (server, path, options) => {
```

Comment:
Not for route files.  They do not use dependency injection and thus must use the parameter order as specified in [Restify Router Magic](https://github.com/tests-always-included/restify-router-magic#route-files) - just take a peek at the Route Files and later in that section where it talks about factory functions.